UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/RadioNavigationalWarnings/ShowRadioNavigationalWarnings.cshtml

```diff
@@ -0,0 +1,83 @@
+﻿@model IEnumerable<UKHO.MaritimeSafetyInformation.Common.Models.RadioNavigationalWarning.RadioNavigationalWarningsData>
+
+@{
+    Layout = "~/Views/Shared/_MSILayout.cshtml";
+}
+
+@{
+    ViewData["Header"] = "Radio Navigation Warnings";
+}
+
+<div id="rnwInfo" class="alert alert-primary p-4" role="alert">
+    <i class="bi bi-exclamation-circle-fill" aria-hidden="true"></i>
+    <div class="alert-head">Important information</div>
+    <p>The following NAVAREA 1 and UK Coastal Warnings (WZs) were in force at: 141009 UTC April 22</p>
+</div>
+
+<!--tabs-->
+<ul class="nav nav-tabs msi-tabs" id="rnwTab" role="tablist">
+    <li class="nav-item" role="presentation">
+        <button class="nav-link active" id="allwarnings-tab" data-bs-toggle="tab" data-bs-target="#allwarnings" type="button" role="tab" aria-controls="allwarnings" aria-selected="true">All warnings</button>
+    </li>
+    <li class="nav-item" role="presentation">
+        <button class="nav-link" id="NAVAREA1-tab" data-bs-toggle="tab" data-bs-target="#NAVAREA1" type="button" role="tab" aria-controls="NAVAREA1" aria-selected="false" tabindex="-1">NAVAREA 1</button>
+    </li>
+    <li class="nav-item" role="presentation">
+        <button class="nav-link" id="ukcostal-tab" data-bs-toggle="tab" data-bs-target="#ukcostal" type="button" role="tab" aria-controls="ukcostal" aria-selected="false" tabindex="-1">UK Costal</button>
+    </li>
```

Comment:
I'm assuming, when the future PBIs are implemented to filter by `dbo.WarningType` that these and the tab panel divs at the bottom will be data driven.  Is that right @usha13529 ?

Reply:
Yes @barrie-cooper, initial idea was that only. But considering the filter "PBI 45749 - Filter the Nav Warnings by area" priority is now lower and can be picked up after Jun, better to not add any navigation for now. 
What are your thoughts on it?

Reply:
Yes @barrie-cooper , now i removed the navigation from the code. It's similar in look to the about menu panel.

Reply:
That's fine, just making sure we're all thinking the same.